extensions-android/README.md

```diff
@@ -46,7 +46,7 @@ executable, e.g. on Linux:
 
 Then run it with:
 
-    ./build.sh
+    ./build.sh <version> <arch> <mode>
```

Comment:
Better support these parameters as optional, e.g. make default values in build.sh.